src/woo_publications/contrib/gpp_zoeken/client.py

```diff
@@ -53,6 +58,20 @@ def get_publication_onderwerpen(
     ]
 
 
+def get_publication_identifiers(
+    publication: Publication,
+) -> list[Kenmerk]:
+    qs = publication.publicationidentifier_set.all().values("kenmerk", "bron")  # pyright: ignore[reportAttributeAccessIssue]
+    return [{"kenmerk": item["kenmerk"], "bron": item["bron"]} for item in qs]
```

Comment:
```suggestion
    qs = publication.publicationidentifier_set.values("kenmerk", "bron")  # pyright: ignore[reportAttributeAccessIssue]
    return [{"kenmerk": item["kenmerk"], "bron": item["bron"]} for item in qs]
```
the `.all()` does not add anything